config.example.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var config = {}
+config.session = {}
```

Comment:
The session secret is not set from within the code, it's set in https://github.com/dnsimple/hello-domains-node/blob/master/config.example.js - does the config loader automatically handle creation of nested objects then? Was the example breaking before you added this?